nimbus.fml.yaml

```diff
@@ -351,3 +380,20 @@ types:
           description: Jump back in onboarding CFR.
         first-run-onboarding:
           description: First run onboarding page updates.
+    MR2022Section:
+      description: The identifiers for the sections of the MR 2022.
+      variants:
+        home-onboarding-dialog-existing-users:
+          description: Home onboarding dialog for upgraded users.
+        home-onboarding-dialog-new-users:
+          description: Home onboarding dialog for new users.
+        sync-cfr:
+          description: CFR for the first time you see a synced tab on the home screen.
+        wallpapers-selection-tool:
+          description: Wallpapers selection dialog tool for the home screen.
+        jump-back-in-cfr:
```

Comment:
I am wondering what are we doing about the equivalent experiments listed under `OnboardingSection`?

Reply:
It's because we want to have a big experiment that allows to configure everything related to the MR, unfortunately there is a Nimbus limitation that doesn't allow to test multiple features in one, for this reason we have to put everything under one single experiment.

Reply:
Should we remove the experiment configurations added before then? This would be everything in `OnboardingSection `

Reply:
In the grand scheme of things, I imagine we want to make sure we aren't using `OnboardingSection` and migrate over to using the experimenets added under this new section

Reply:
We will create a follow up ticket  to remove the existing features and its sections.

Reply:
For now, we want to land this and uplift it to 105 then on 106 nightly we can remove it.

Reply:
Filed https://github.com/mozilla-mobile/fenix/issues/26620